builder/vmware/common/driver_fusion5.go

```diff
@@ -172,9 +169,9 @@ func (d *Fusion5Driver) Verify() error {
 	d.VmwareDriver.NetworkMapper = func() (NetworkNameMapper, error) {
 		pathNetworking := filepath.Join(libpath, "networking")
 		if _, err := os.Stat(pathNetworking); err != nil {
-			return nil, fmt.Errorf("Could not find networking conf file: %s", pathNetworking)
+			return nil, fmt.Errorf("error finding networking configuration file: %s", pathNetworking)
```

Comment:
I'd suggest including the error message here for troubleshooting purposes
```suggestion
			return nil, fmt.Errorf("error finding networking configuration file %q: %s", pathNetworking, err)
```

Reply:
Resolved in 67054cd.